src/pages/authPages/RegistrationPage.tsx

```diff
@@ -1,8 +1,66 @@
 import React, { FC } from 'react';
 import { Box, TextField, Button, Typography, Container, Paper } from '@mui/material';
 import NavigationBar from '../homePage/NavigationBar';
+import { useAuthContext } from '../../context/AuthContext';
+import { useNavigate } from 'react-router-dom';
+import { Controller, useForm } from 'react-hook-form';
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
 
+
+
+interface RegisterData {
+    name: string;
+    lastName: string;
+    email: string;
+    password: string;
+
+}
+
+type Inputs = {
+    name: string,
+    lastName: string,
+    email: string,
+    password: string,
+    confirm_password: string,
```

Comment:
od kiedy uzywamy snake case'a?
```suggestion
    confirmPassword: string,
```

Reply:
to przez tego ch*jka copilot'a 😂 moje przeocznie i nie nie używamy 🐍 case